7/Dockerfile

```diff
@@ -1,46 +1,47 @@
-FROM centos:centos7
+FROM centos/s2i-base-centos7
 
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+EXPOSE 8080
 
```

Comment:
Thinking, is the expose statement necessary? Wouldn't it be better to leave opening ports for the user build image?
In golang image I have not exposed any port as I think that there is not standard port to open for generic Go app.